pkgs/development/python-modules/treeo/default.nix

```diff
@@ -8,20 +9,26 @@
 
 buildPythonPackage rec {
   pname = "treeo";
-  version = "0.4.0";
+  # Note that there is a version 0.4.0, but it was released in error. At the
+  # time of writing (2022-03-29), v0.0.11 is the latest as reported on GitHub
+  # and PyPI.
+  version = "0.0.11";
   format = "pyproject";
 
   src = fetchFromGitHub {
     owner = "cgarciae";
     repo = pname;
     rev = version;
-    sha256 = "176r1kgsdlylvdrxmhnzni81p8m9cfnsn4wwn6fnmsgam2qbp76j";
+    hash = "sha256-zs3F8i+G5OX/A9wOO60xVuvnm2QqrL+dHIrC0qwH37o=";
   };
 
-  postPatch = ''
-    substituteInPlace pyproject.toml \
-      --replace 'typing-extensions = "^3.10.0"' 'typing-extensions = "*"'
-  '';
+  # See https://github.com/cgarciae/treex/issues/68.
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/cgarciae/treeo/pull/14.patch";
```

Comment:
Please fetch the commits individually to prevent errors when the PR is updated.

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/166167/commits/db60f460b67463cdc12c3ad2982a49a80269717e